news/39.bugfix.3

```diff
@@ -0,0 +1 @@
+Do not export parent info.  This information is not needed.  @mauritsvanrees
```

Comment:
There's a slight change in behavior during imports, which is that parents are now always found by path and not by UID. I think that's okay but we should mention it.

Reply:
I have updated the news snippet.